### DISCUSSION TITLE
Hardcoded Icon in Breadcrumb

### DISCUSSION THREAD

**Discussion by eissmann** [public-ui/kolibri#6780] [Q&A] [answered]
Sollte dieses Trenner-Icon nicht besser konfigurierbar sein und lediglich als Fallback die Codicons verwenden? So wird die Zusammenarbeit mit einer Designagentur o.ä. sehr schwierig.

https://github.com/public-ui/kolibri/blob/41538a9c4cd274d6d8023da93e411ca63697e8ac/packages/components/src/components/breadcrumb/shadow.tsx#L23

**Comment by deleonio** [comment, marked as answer]
Hallo @eissmann,

vielen Dank für diese Nachfrage. Das ist auf den ersten Blick vollkommen korrekt. Jedoch wird es durch das Theming ermöglicht, auf diese CSS-Klassen mit anderen Icons zu reagieren ;-)

Hier ist dazu ein Blog-Beitrag: https://public-ui.github.io/blog/2023/05/12

Hier ein Beispiel-Code: https://github.com/public-ui/kolibri/blob/41538a9c4cd274d6d8023da93e411ca63697e8ac/packages/themes/bmf/src/components/breadcrumb.scss#L16
